sonic-net/sonic-buildimage
# [minigraph] Add rack_mgmt_rack parse support in minigraph.py

#### Why I did it
We need to store information of power shelf in config_db for SONiC MX switch. Current minigraph parser cannot parse rack_mgmt_map field.

##### Work item tracking
- Microsoft ADO **(number only)**: 22179645

#### How I did it
Add support for parsing rack_mgmt_map.

#### How to verify it
1. Run `config load_minigraph` cmd to verify.
2. Unit test
```
test_minigraph_rack_mgmt_map (tests.test_minigraph_case.TestCfgGenCaseInsensitive) ... ok
```



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
